pypeit/spectrographs/not_alfosc.py

```diff
@@ -0,0 +1,222 @@
+from pypeit.par import pypeitpar
+from pypeit.spectrographs import spectrograph
+from pypeit.core import parse
+from pypeit.images import detector_container
+
+from IPython import embed
+
+
+class NOTALFOSCSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle NOT ALFOSC spectrograph
+    """
+    ndet = 1
+
+    def __init__(self):
+        # Get it started
+        super(NOTALFOSCSpectrograph, self).__init__()
+        self.spectrograph = 'not_alfosc'
+        self.telescope = telescopes.NOTTelescopePar()
+
+    def get_detector_par(self, hdu, det):
+        """
+        Return a DectectorContainer for the current image
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`):
+                HDUList of the image of interest.
+                Ought to be the raw file, or else..
+            det (int):
+
+        Returns:
+            :class:`pypeit.images.detector_container.DetectorContainer`:
+
+        """
+
+        # http://www.not.iac.es/instruments/detectors/CCD14/
+
+        # Detector 1
+        detector_dict = dict(
+            binning         =self.get_meta_value(self.get_headarr(hdu), 'binning'),
+            det             = 1,
+            dataext         = 1,
+            specaxis        = 0,
+            specflip        = True,
+            spatflip        = False,
+            xgap            = 0.,
+            ygap            = 0.,
+            ysize           = 1.,
+            platescale      = 0.2138,
+            mincounts       = -1e10,
+            darkcurr        = 1.3,   # e-/pix/hr
+            saturation      = 700000., # ADU
+            nonlinear       = 0.86,
+            datasec         = np.atleast_1d('[:,{}:{}]'.format(1, 2062)),  # Unbinned
+            oscansec        = None,
+            numamplifiers   = 1,
+        )
+
+        # Parse datasec, oscancsec from the header
+        head1 = hdu[1].header
+        detector_dict['gain'] = np.atleast_1d(head1['GAIN'])  # e-/ADU
+        detector_dict['ronoise'] = np.atleast_1d(head1['RDNOISE'])  # e-
+
+        # Return
+        return detector_container.DetectorContainer(**detector_dict)
+
+    def default_pypeit_par(self):
+        """
+        Set default parameters for reductions.
+        """
+        par = pypeitpar.PypeItPar()
+        par['rdx']['spectrograph'] = 'not_alfosc'
+
+        # Ignore PCA
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+
+        # Set pixel flat combination method
+        par['calibrations']['pixelflatframe']['process']['combine'] = 'median'
+        par['calibrations']['pixelflatframe']['process']['sig_lohi'] = [10.,10.]
+        # Wavelength calibration methods
+        #par['calibrations']['wavelengths']['method'] = 'holy-grail'
+        par['calibrations']['wavelengths']['method'] = 'full_template'
+        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'NeI']
+        #par['calibrations']['wavelengths']['reid_arxiv'] = 'mdm_osmos_mdm4k.fits'
```

Comment:
I suspect this filename is not intended?

Reply:
removed.